infra/deploy.sh

```diff
@@ -377,7 +382,7 @@ checkSKUQuotas() {
     printf "Checking Location for SKU Quota Usage... "
     local location=$1
     local vm_usage_report=$(
-        az vm list-usage --location $location
+        az vm list-usage --location $location -o json
```

Comment:
When calling the azure cli, we want to always be explicit and specify the `-o json` flag. This guarantees that the output is in json format. Power users of azure cli may have their output format set to something other than `json`.